core/controllers/reader.py

```diff
@@ -1016,3 +1013,25 @@ def post(self, exploration_id):
             self.payload.get('report_text'),
             self.user_id)
         self.render_json(self.values)
+
+
+class ExplorationFeaturesHandler(base.BaseHandler):
+    """Returns features the given exploration is configured to support."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_play_exploration
```

Comment:
This would change the permissions for the whitelist right? Are we missing any situation by not allowing open access? Just confirming.